src/sentry/integrations/discord/utils/consts.py

```diff
@@ -0,0 +1,249 @@
+    170006: "Sticker frame rate is either too small or too large",
+    170007: "Sticker animation duration exceeds maximum of 5 seconds",
+    180000: "Cannot update a finished event",
+    180002: "Failed to create stage needed for stage event",
+    200000: "Message was blocked by automatic moderation",
+    200001: "Title was blocked by automatic moderation",
+    220001: "Webhooks posted to forum channels must have a thread_name or thread_id",
+    220002: "Webhooks posted to forum channels cannot have both a thread_name and thread_id",
+    220003: "Webhooks can only create threads in forum channels",
+    220004: "Webhook services cannot be used in forum channels",
+    240000: "Message blocked by harmful links filter",
+    350000: "Cannot enable onboarding, requirements are not met",
+    350001: "Cannot update onboarding while below requirements",
+}
+
+"""
+After an alert rule is created, the user can perform actions that can break the alert and
+produce error Discord API responses.
+
+For our SLO that monitors Discord response status codes, we _do_ want to keep track of
+responses that indicate a problem from our end i.e. 50109
+(the request body contains invalid JSON.).
+We _don't_ want to keep track of responses that are failing because of a user config change.
+
+In order to make the SLO more relevant, we want to ignore specific discord provided codes that
+can be returned when a user changes a configuration and breaks the alert.
+
+Complete list of codes: https://discord.com/developers/docs/topics/opcodes-and-status-codes#http
+
+Here is a working list of codes that we should ignore and why:
+"""
+DISCORD_USER_ERRORS = {
+    10001,  # user deleted account
+    10002,  # user deleted application
+    10003,  # user deleted channel
+    10004,  # user deleted guild
+    10005,  # user deleted integration
+    10008,  # user deleted message
+    10011,  # user deleted role
+    10013,  # user deleted account
+    10087,  # user deleted tag
+    20009,  # user's error message contains explicit content
+    20018,  # user changed action permissions to owner-restricted
+    50001,  # user changed bot's permissions
+    50007,  # user is preventing direct messages from bot **direct messages not implemented yet
+    50009,  # bot not added to channel yet
+    50013,  # user changed bot's permissions for actions
+    50055,  # user changed or deleted guild (would likely only hit 10003)
+    50083,  # user archived thread (would likely only hit 10003)
+    50178,  # user not verified yet
+    60003,  # user added 2 factor not sure if this can happen with bot
+    80004,  # user DiscordTag does not exist (could have been deleted or modified)
+    160002,  # user removed bot's permission to reply to thread
+    160005,  # user locked a thread
+    200000,  # bot's message blocked by auto mod
+    200001,  # bot's message title blocked by auto mod
+    240000,  # auto mod blocked because message contained harmful link
```

Comment:
i think this would be on us no? The auto mod blocked us because our bot sent something that we shouldn't have. I don't think the user has any control over this?

Reply:
@ykamo001 iirc, we use customer provided data to generate our messages. In the extremely rare case that a user's error message contains content that the auto mod would block, I imagine that should be a 'user-related' issue. 

The titles we generate don't use any user provided data so I can remove that one.

Reply:
this is probably an edge case that's rare so probably okay to consider it user error, and we can investigate appropriately